mappings/net/minecraft/nbt/AbstractNbtList.mapping

```diff
@@ -0,0 +1,20 @@
+CLASS net/minecraft/class_2483 net/minecraft/nbt/AbstractNbtList
+	COMMENT Represents an abstraction of a mutable NBT list which holds elements of the same type.
+	METHOD add (ILjava/lang/Object;)V
+		ARG 1 index
+		ARG 2 value
+	METHOD method_10533 addElement (ILnet/minecraft/class_2520;)Z
+		ARG 1 index
+		ARG 2 element
+	METHOD method_10535 setElement (ILnet/minecraft/class_2520;)Z
+		ARG 1 index
+		ARG 2 element
+	METHOD method_10601 getElementType ()B
+		COMMENT Gets the type of elements held in this list.
+		COMMENT
+		COMMENT @return the type of elements
```

Comment:
This wording is a bit confusing and to me suggests that a list can hold different types, so I would rename it to `getType`:

```suggestion
	METHOD method_10601 getType ()B
		COMMENT Gets the type of element that this list holds.
		COMMENT
		COMMENT @return the type of element that this list holds
```

Reply:
`getType` is a big no as it's already used in `NbtElement` with the same signature.

Reply:
Would `getListType` work instead?

```suggestion
	METHOD method_10601 getListType ()B
		COMMENT Gets the {@linkplain NbtElement#getType type} of element that this list holds.
		COMMENT
		COMMENT @return the type of element that this list holds
```

Reply:
yes